data/yaml/venues.yaml

```diff
@@ -238,6 +238,10 @@ conll:
   is_toplevel: true
   name: Conference on Computational Natural Language Learning
   oldstyle_letter: K
+constraint:
+  acronym: CONSTRAINT
+  name: Workshop on Combating Online Hostile Posts \\in Regional Languages during
+    Emergency Situation
 corbon:
```

Comment:
I spoke with them and decided we can use this acronym. @xinru1414, can you fix the `\\`, and add the workshop URL: https://lcs2.iiitd.edu.in/CONSTRAINT-2022/